src/current/_includes/v25.2/leader-leases-node-heartbeat-use-cases.md

```diff
@@ -0,0 +1,7 @@
+For the purposes of [Raft replication]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) and determining the [leaseholder]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder) of a [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range), node health is no longer determined by heartbeating a single "liveness range"; instead it is determined using [Leader leases]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leader-leases).
```

Comment:
Is this new in v25.2 specifically? If so, add the "New in" verbiage here?

Reply:
yes! added the 'new in' tag thingy